tests/fixtures/long_names/v1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "version": "long_survey_name__the_quick__brown_fox_jumps_over_the_lazy_dog_v1",
+  "content": {
+    "survey": [
+      {
+        "type": "begin repeat",
```

Comment:
formpack's new preferred alias here for groups, repeats, etc have underscores: `begin_repeat` and `end_repeat`.

It's corrected in `replace_aliases`